hack/components/bump-linux-bridge.sh

```diff
@@ -22,7 +22,7 @@ echo 'Get linux-bridge tag'
 LINUX_BRIDGE_TAG=$(git-utils::get_component_tag ${LINUX_BRIDGE_PATH})
 
 echo 'Build container image with linux-bridge binaries'
-LINUX_BRIDGE_TAR_CONTAINER_DIR=/usr/src/github.com/containernetworking/plugins/bin
+LINUX_BRIDGE_TAR_CONTAINER_DIR=/usr/src/containernetworking/plugins/bin
```

Comment:
It this fixing the first part of the problem? eg:
```
One, we never installed our own bridge and tuning CNI, since the source
of our copy was not matching directories used in the CNI plugins
container image. The only reason why this worked so far is that we were
able to consume these plugins from the Multus deployment.
```

Reply:
I wonder though whether the result is better then. We would ship our own CNIs, but when Multus gets installed, we will never know which one is being used. I'm worried that it would make debugging harder.

I'm open to suggestions on how to fix this for good